src/2023/index.html

```diff
@@ -14,6 +14,9 @@ <h1 class="main-title">Helvetic Ruby 2023</h1>
           <p class="lead">A Ruby conference in Switzerland.</p>
           <p class="venue-date"><time>24.11.2023</time> in Bern, Switzerland</p>
 
+          <a href="mailto: keep-me-posted@helvetic-ruby.ch?subject=Keep%20me%20posted&body=[You%20can%20leave%20this%20part%20of%20the%20mail]" class="small shy-white-link">
```

Comment:
This text feels a little too close to the venue information to me. I would add more space above it. What do you think?

Reply:
You're right. Added some vertical space:

![Screenshot 2023-02-14 at 11 57 36](https://user-images.githubusercontent.com/1409672/218715823-e352962d-e649-4d3c-9e11-8847b54a4589.png)